### PR TITLE
you missed some .

### PR DESCRIPTION
**Pull request by Matsko3**
## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [x] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ EN | [中文](README_zh-CN.md)
     
 2. Set network proxy to `127.0.0.1:8080` or the proxy port you specified.
 
-**you can also use `start.cmd` to start servers and proxy daemons automatically**
+**you can also use `start.cmd` to start servers and proxy daemons automatically.**
```
Comment (Birdulon) [inline README.md]:
```suggestion
**You can also use `start.cmd` to start servers and proxy daemons automatically.**
```

Comment (Matsko3) [reply README.md]: So true

**Comment by KiminaNatsuki** [comment]
ayy lmao

File: README.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ There is a dummy user named "Server" in every player's friends list that you can
  
 # Quick Troubleshooting
 
-* If compiling wasn't successful, please check your JDK installation (JDK 17 and validated JDK's bin PATH variable)
+* If compiling wasn't successful, please check your JDK installation (JDK 17 and validated JDK's bin PATH variable.)
```
Comment (lunaticwhat) [inline README.md:155]: why the . is
inside those brackets

Comment (KiminaNatsuki) [reply README.md:155]: this is a joke pr lmao

Comment (KiminaNatsuki) [reply README.md:155]: add some . to get contributor 100% works

Comment (lunaticwhat) [reply README.md:155]: > add some . to get contributor 100% works

can actually be a contributor if the dot was actually in its right place

Comment (alt3ri) [reply README.md:155]: . moment

**Comment by Matsko3** [comment]
.

**Comment by Arikatsu** [comment]
.

**Comment by KiminaNatsuki** [comment]
begone

**Comment by Matsko3** [comment]
> begone

.

File: README.md (hunk 2 above)
Comment (KiminaNatsuki) [reply README.md:155]: def not resolved

**Comment by M9GG** [comment]
Tell me dadi🥺

**Comment by ghost** [comment]
L

**Comment by lilmayofuksu** [comment]
> ayy lmao

.

**Comment by lunaticwhat** [comment]
> > ayy lmao
> 
> .

so true

**Pull request closed without merging**